backend/amd64/proc.ml

```diff
@@ -646,6 +650,45 @@ let frame_required ~fun_contains_calls ~fun_num_stack_slots =
 let prologue_required ~fun_contains_calls ~fun_num_stack_slots =
   frame_required ~fun_contains_calls ~fun_num_stack_slots
 
+(* returned size includes return address *)
+let frame_size ~stack_offset ~fun_contains_calls ~fun_num_stack_slots =
```

Comment:
Could we maybe factorize this code with the function of the same name in `emit.mlp`, and likewise for the `slot_offset` function?

Reply:
Maybe as a second PR.  @xclerc what do you think?

Reply:
I would second @Ekdohibs motion in
principle, but upon closer examination
it is not as simple as moving the code
from `Emit` (the version from `Emit`
relies on a `ref` for instance).

Since we want to merge it soon, maybe
just a CR summarizing @Ekdohibs'
intentions?

Reply:
(And we should indeed do that
before we add new register/stack
classes.)